generator/internal/genclient/parser/protobuf/protobuf.go

```diff
@@ -197,6 +197,7 @@ func MakeAPI(serviceConfig *serviceconfig.Service, req *pluginpb.CodeGeneratorRe
 		ServiceByID: make(map[string]*genclient.Service),
 		MessageByID: make(map[string]*genclient.Message),
 		EnumByID:    make(map[string]*genclient.Enum),
+		Package:     req.GetSourceFileDescriptors()[0].GetPackage(),
```

Comment:
Wouldn't this crash if there are no `GetSourceFileDescriptors()` ?  

And what if there are multiple package names?

Reply:
This is guaranteed to be set, assuming we sent in some protos. Added a comment to explain this.